tests/helper.js

```diff
@@ -8,10 +11,9 @@
   exports.apiProduct = {guid: 'server-guid', name: "Name from API",
                         logo_url: "img.jpg"};
 
-
   exports.setUp = function setUp() {
     exports.server = sinon.fakeServer.create();
-    exports.settings = fxpay.settings.configure({
+    exports.settings = settings.configure({
```

Comment:
I'm not really sure how useful this is. You could alternatively replace ``helper.settings`` with required settings in each test. But no biggie.